loopdb/store_test.go

```diff
@@ -29,6 +30,26 @@ var (
 		1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 3,
 	}
 
+	senderInternalKey = [33]byte{
```

Comment:
just as a comment: I think we ought to clearly specify if we mean pubkey or privkey, as here it is pubkey, where as in the bolt db bucket it ist the privkey. I would vouch to always specify wether pubkey or privkey is meant